test/browser_tests/cucumber/step_definitions/global-search.js

```diff
@@ -121,9 +121,11 @@ Then( 'I should navigate to search portal',
     await browser.wait( EC.visibilityOf( _dom.searchBtn ) );
     await _dom.searchBtn.click();
 
-    return expect( browser.getCurrentUrl() )
+    return expect( browser.getCurrentUrl().then( url => {
+      url.substring( 0, url.lastIndexOf( '&' ) )
```

Comment:
Search URL now has a `_gl` parameter (Google Analytics maybe?), which changes, so we strip it off here.